src/ansys/fluent/core/session_solver.pyi

```diff
@@ -1,17 +1,17 @@
 from ansys.fluent.core.datamodel_231.preferences import Root as preferences_root
 from ansys.fluent.core.datamodel_231.solverworkflow import Root as solverworkflow_root
 from ansys.fluent.core.datamodel_231.workflow import Root as workflow_root
-from ansys.fluent.core.solver.settings_231 import (
+from ansys.fluent.core.solver.settings_231.current_parametric_study import (
     current_parametric_study,
-    file,
```

Comment:
The type-hint should be a class instead of a module. Although the previous version was working around the previous version of settingsgen code.